include/fg/image.h

```diff
@@ -114,7 +114,7 @@ FGAPI fg_err fg_get_image_type(fg_dtype* pOut, const fg_image pImage);
 
    \return \ref fg_err error code
  */
-FGAPI fg_err fg_get_image_pbo(unsigned* pOut, const fg_image pImage);
+FGAPI fg_err fg_get_image_resource_id(unsigned* pOut, const fg_image pImage);
```

Comment:
Okay, changing c version to `pixel_buffer` and c++ as `pixels` similar to how others are 
`vertex_buffer` / `vertices`
 `color_buffer` / `colors` 
 `alpha_buffer` / `alphas` 
are named. is that fine ?